App/src/pages/CourseBrowser.tsx

```diff
@@ -1,30 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import Select from "react-select";
 import { Box, Button, Container, FormLabel, Heading, Stack } from "@chakra-ui/react";
-
+import { useNavigate } from "react-router-dom";
+import { Input } from "@chakra-ui/react";
 interface SubjectItem {
     value: string;
     label: string;
 }
 
+// when the api is implemented I will make the subjects available in the drop down
```

Comment:
The API endpoint for this is actually implemented, it's the course settings endpoint. It's already being used in some spots in the frontend, so just see how they call the existing function.

Reply:
done in [7b0862d](https://github.com/JamesPartsafas/ConcordiaCurriculumManager/pull/230/commits/7b0862d9e483151f91d880ddf2c8ffbfe23f5ae5)